hil-test/tests/esp_radio_init.rs

```diff
@@ -119,4 +119,24 @@ mod tests {
 
         _ = esp_radio::wifi::new(&esp_radio_ctrl, peripherals.WIFI).unwrap();
     }
+
+    #[test]
```

Comment:
We don't even need this test to run, just to compile, right? For that you can turn this into something like [this](https://github.com/esp-rs/esp-hal/blob/36634a4f9283ea4cc6b38a5598aa67af025cef5b/hil-test/tests/esp_radio_ble_controller.rs#L25). We can sort out the panic separately from this PR.

Reply:
Am I supposed to do something with this PR or not needed?